app/utils/callback/lava.py

```diff
@@ -495,9 +419,7 @@ def __init__(self, job_data, definition_meta, lab_name,
                                         Loader=yaml.CLoader)
             self.log = self._prepare_log(job_data["log"])
         except yaml.YAMLError:
-            ret_code = 401
-            msg = "Invalid test data from LAVA callback"
-            utils.errors.add_error(self._errors, ret_code, msg)
+            raise CallbackParseError
```

Comment:
Might be worth adding an error message to make it clear it's a YAML syntax error - or let it raise a `YAMLError` if it's always going to be due to this.

Reply:
Changed to plain yaml.YamlError